### ISSUE TITLE
How to verify structure a neural network in keras model?

### ISSUE THREAD

**Issue by Denis-Evseev** [keras-team/keras#7743]
I'm new in Keras and Neural Networks. I'm writing a thesis and trying to create a SimpleRNN in Keras as it is illustrated below:

[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/VwPWx.jpg

As it is shown in the picture, I need to have 4 inputs + 2 outputs and can have any number of neurons in the hidden layer.

This is my code:

    model = Sequential()

    model.add(SimpleRNN(4, input_shape=(1, 4), activation='sigmoid', return_sequences=True))
    model.add(Dense(2))
    model.compile(loss='mean_absolute_error', optimizer='adam')
    model.fit(data, target, epochs=5000, batch_size=1, verbose=2)

    predict = model.predict(data)

1) Does my model implement the graph?   
2) Is it possible to specify connections between Input and Hidden layers or Output and Input layers?  

Can anybody help?
